### PR TITLE
Add parser for Slovakia

### PR DESCRIPTION
**Pull request by ivorbosloper**

### CODE REVIEW

File: fiboa_cli/datasets/sk.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import zipfile
+import os
+from ..convert_utils import convert as convert_, download_files
```
Comment (m-mohr) [inline fiboa_cli/datasets/sk.py:3]:
```suggestion
from ..convert_utils import convert as convert_
```

File: fiboa_cli/datasets/sk.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+SOURCES = {
+#    "https://data.slovensko.sk/download?id=ed0b4a21-8774-4bb1-aab3-fea4ddab9010&blocksize=0": "DPB2024_20240912.zip"
+    "https://data.slovensko.sk/download?id=e39ad227-1899-4cff-b7c8-734f90aa0b59&blocksize=0": "HU2024_20240917.zip"
```
Comment (m-mohr) [inline fiboa_cli/datasets/sk.py:7]: "https://data.slovensko.sk/download?id=e39ad227-1899-4cff-b7c8-734f90aa0b59&blocksize=0": [
        "HU2024_20240917shp/HU2024_20240917.shp"
    ]

File: fiboa_cli/datasets/sk.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+def convert(output_file, cache = None, **kwargs):
+    # The zipfile has an embedded directory. This already fails at `gpd.list_layers(path)`
+    # Workaround: download + unzip, use unzipped shapefile path as source-place
+    assert cache, "Cache is required for this parser"
+    download_files(SOURCES, cache)
+    path = next(iter(SOURCES.values()))
+    with zipfile.ZipFile(os.path.join(cache, path), 'r') as zip_file:
+        zip_file.extractall(cache)
+
```
Comment (m-mohr) [inline fiboa_cli/datasets/sk.py:64]:
```suggestion
```

File: fiboa_cli/datasets/sk.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    convert_(
+        output_file,
+        cache,
+        os.path.join(cache, path.replace('.zip', 'shp'), path.replace('.zip', '.shp')),
```
Comment (m-mohr) [inline fiboa_cli/datasets/sk.py:68]:
```suggestion
        SOURCES,
```

**Review by m-mohr — CHANGES_REQUESTED** [review]
The converter can extract the files for you, see comments.

**Comment by ivorbosloper** [comment]
Thanks, I wasn't aware of the list-values option for SOURCES . Way better like this. We can merge @m-mohr

**Review by m-mohr — APPROVED** [review]

**Pull request merged**